controls/KEYC-01-000005.rb

```diff
@@ -1,31 +1,48 @@
 # -*- encoding : utf-8 -*-
+
 control "KEYC-01-000005" do
   title "Keycloak must be configured to automatically audit account creation."
   desc  "Once an attacker establishes access to a system, the attacker often attempts to create a persistent method of reestablishing access. One way to accomplish this is for the attacker to simply create a new account. Auditing of account creation is one method for mitigating this risk. A comprehensive account management process will ensure an audit trail documents the creation of user accounts and, as required, notifies administrators and/or managers. Such a process greatly reduces the risk that accounts will be surreptitiously created and provides logging that can be used for forensic purposes."
   desc  "rationale", ""
   desc  "check", "
-    If Keycloak relies on directory services for user account management, this is not applicable and the connected directory services must perform this function. 
+    If Keycloak relies on directory services for user account management, this is not applicable and the connected directory services must perform this function.
     
     Verify Keycloak is configured to automatically audit account creation.
     
     If Keycloak is not configured to automatically audit account creation, this is a finding.
     
-    To check if Keycloak is configured to audit account creation, you can run the following commands from a privileged account on the Keycloak admin CLI:
+    To check if Keycloak is configured to audit account creation, log into the Keycloak admin CLI with a privileged account:
```

Comment:
Are these changes to the metadata from pulling a new version of the XCCDF file from the vulcan project?

Reply:
Yes. Prior push was before changes in Vulcan.